src/core/libmaven/isotopeDetection.cpp

```diff
@@ -298,10 +306,13 @@ void IsotopeDetection::childStatistics(
     child.parent = parentgroup;
     child.setType(PeakGroup::Isotope);
     child.groupStatistics();
-    if (_mavenParameters->clsf->hasModel()) {
+
+   /* if (_mavenParameters->clsf->hasModel())
+     {
         _mavenParameters->clsf->classify(&child);
         child.groupStatistics();
-    }
+       // cerr<<"......................You are in Child Statisitcs"<<endl;
+    } */
```

Comment:
If this comment has no informative purpose, please remove this block entirely. Otherwise add details on what this commented block helps illustrate.